src/modules/Builder/features/ActivityItems/ItemConfiguration/__tests__/ItemConfigurationSliderRows.test.tsx

```diff
@@ -0,0 +1,597 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-nocheck
+import { createRef } from 'react';
+import { fireEvent, screen, waitFor } from '@testing-library/react';
+
+import { CHANGE_DEBOUNCE_VALUE, ItemResponseType } from 'shared/consts';
+import { asyncTimeout, renderWithAppletFormData } from 'shared/utils';
+
+import {
+  getAppletFormDataWithItem,
+  mockedAlertsTestid,
+  mockedItemName,
+  mockedTextInputOptionTestid,
+  mockedUseParams,
+  renderItemConfiguration,
+  setItemConfigSetting,
+  setItemResponseType,
+} from '../__mocks__';
+import { ItemConfigurationSettings } from '../ItemConfiguration.types';
+
+jest.mock('react-router-dom', () => ({
```

Comment:
can y move pls router mock to separate func?

Reply:
removed at all as needless mock